components/browser_kit.rst

```diff
@@ -222,8 +222,9 @@ also delete all the cookies::
     $client = new Client();
     $client->request('GET', 'http://symfony.com');
 
-    // delete history
+    // reinitializes the browser state (history and cookies are deleted)
     $client->restart();
+    // which internally calls history->clear() and cookieJar->clear()
```

Comment:
Not sure about this line:

```
which internally calls history->clear() and cookieJar->clear()
```

It explains an internal coding details (which makes docs hard to maintain). And we already explain above that this clears cookies and history, so maybe it's unneeded. Thanks!